qtconsole/jupyter_widget.py

```diff
@@ -277,13 +276,10 @@ def _handle_kernel_info_reply(self, rep):
             super(JupyterWidget, self)._started_channels()
 
     def _started_channels(self):
-        """Reimplemented to make a history request and load %guiref."""
+        """Reimplemented to make a history request"""
```

Comment:
Perhaps a semantic quibble on my part but "reimplemented" feels not quite accurate to me. BaseFrontendWidget is abstract; FrontEndWidget implements the method from the abstract BaseFrontEndWidget; JupyterWidget subclasses from FrontEndWidget.

The docstrings for each method in this section are not consistent. "Override method" or "Redefine method" feel more accurate to me. Now that I'm thinking about it, it may just be best to add a comment here:
```
 #---------------------------------------------------------------------------
 # 'BaseFrontendMixin' abstract interface
 #
 # For JupyterWidget,  override FrontendWidget methods which implement the
 # BaseFrontend Mixin abstract interface
 #---------------------------------------------------------------------------
 ```